tox.ini

```diff
@@ -1,15 +1,25 @@
 [tox]
-envlist = py3, flake8, pycodestyle
+envlist = py3, flake8, flake8-{trusty,xenial,bionic}
+
+# N.B. If this needs to be expanded to include a third type of test, a
+# "new-test: new-test-commands" line should be added to the commands stanza
+# (and perhaps the deps stanza, if additional dependencies are required), and
+# the default "!flake8" line should be extended to "!flake8-!new-test"
 
 [testenv]
 deps =
     -rrequirements.txt
     -rtest-requirements.txt
-commands = nose2 --with-coverage --coverage uaclient/ uaclient
-
-[testenv:flake8]
-commands = flake8 uaclient setup.py
-
-[testenv:pycodestyle]
-commands = pycodestyle {posargs:uaclient setup.py}
+    trusty: -cdev/trusty-constraints.txt
+    xenial: -cdev/xenial-constraints.txt
+    bionic: -cdev/bionic-constraints.txt
```

Comment:
I leave this up to you... but I prefer constraints-{release}.txt so they are all stacked near each other, but that is personal preference

Reply:
Yep, definitely preferable.